sources/web/datalab/polymer/templates/NewNotebookTemplate.ipynb

```diff
@@ -0,0 +1,43 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## New Notebook\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Place your code here.\n",
+    "\n"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
```

Comment:
Do we really want to include a kernel spec here? I think python 2 is the default anyway, but we should consider either taking it out or making it a parameter placeholder.